server/chat-plugins/scavengers.ts

```diff
@@ -652,8 +676,7 @@ export class ScavengerHunt extends Rooms.RoomGame {
 
 		const finalHint = current.number === this.questions.length ? "final " : "";
 
-		player.sendRoom(
-			`|raw|<div class="ladder"><table><tr>` +
+		const questionDisplay = `|raw|<div class="ladder"><table><tr>` +
```

Comment:
There's a ton of unnecessary new lines in this

Reply:
@Zarel wrote this

Reply:
Yes, but that doesn't mean the new lines are necessary? It's a variable that's separately declared now so the line length shouldn't affect it I'm pretty sure.

Reply:
The new lines were necessary when it was still wrapped in the function.

Reply:
I definitely prefer the newlines for readability.

Reply:
I don't like the addition of the `questionDisplay` variable, though. What's the point?